data/starts.txt

```diff
@@ -92,44 +95,45 @@ start "hai"
 	`	You can probably sell the shield regenerator without any consequences, so long as you do not get too close to hostile territory. You may even downgrade the anti-missile turret.`
-	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1 million 431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1.431M credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
 	name
 	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
 	choice
 		`	(Go to the shipyard.)`
 			decline
 		`	(Go see what the captain has to offer first.)`
 	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully they did not yet sell the Grasshopper.`
-	`	The captain greets you in Hai, but you've somehow managed to never really learn the language. Most everyone speaks yours to at least some degree anyway, and once they realize you do not understand, they switch with little effort.`
+	`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned it, due to most Hai speaking your language. As usual, they are able to switch to speaking yours without much difficulties.`
 	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
 	choice
 		`	"Yes indeed."`
 		`	"How did you know?"`
 		`	"Sorry, not interested."`
 			decline
-	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new but still looks in surprisingly good condition.`
+	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
 	scene "thumbnail/hai grasshopper"
-	`	"Because I like you I'll give you a friendly price. What do you say about... one million flat, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
+	`	"Because I like you I'll give you a friendly price. What do you say about... 1.132M credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```

Comment:
Honestly, I prefer the first one, but if you feel like we have to put the exact amount there, this also works. I just don't see anyone actually telling you `what about... 1.132` like it's a number they came up with on the spot. (Also, I'm changing M to million because that's how people say it in the game; M is only used in the outfitter/shipyard iirc.)
```suggestion
	`	"Because I like you I'll give you a friendly price. What do you say about... 1.132 million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```

Reply:
It used to be 1 M I now changed it to be more caus otherwise its too easy
```suggestion
	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```
what about this?

Reply:
Works I guess.